src/Dfe.PlanTech.Web/Views/Shared/Components/CategorySection/SubtopicRecommendation.cshtml

```diff
@@ -0,0 +1,26 @@
+@using Dfe.PlanTech.Domain.Constants
+@model Dfe.PlanTech.Domain.CategorySection.CategorySectionRecommendationDto
+
+
+@if (Model.NoRecommendationFoundErrorMessage != null)
+{
+    <govuk-error-message>@Model.NoRecommendationFoundErrorMessage</govuk-error-message>
+}
+else if (Model.RecommendationSlug != null)
+{
+    if (TempData.ContainsKey("SectionSlug"))
+    {
+        var sectionSlug = TempData["SectionSlug"] as string;
+        @if (!string.IsNullOrEmpty(sectionSlug) && sectionSlug == Model.SectionSlug)
+        {
+            <task-list-tag colour="@TagColour.Yellow">New</task-list-tag>
+        }
+    }
+    <govuk-button-link
+        class="go"
```

Comment:
Is this "go" class meant to be here? Pretty sure I asked before but I don't remember the response.

Reply:
Nope, I must have copied that from somewhere else by accident, that said I can't see it anywhere else so not quite sure why I added that... deleted